deploy/script/katello-backup

```diff
@@ -0,0 +1,80 @@
+#!/bin/bash
+
+# Echo all commands
+usage(){
+  echo "Usage: katello-backup <backup directory>"
+  echo " eg: $ katello-backup /var/tmp/katello-backup/"
+}
+
+if [ "$#" -ne 1 ]; then
+  echo "ERROR: Please specify an export directory"
+  usage
+  exit 1
+fi
+
+# Backup dir
+umask 0027
+BDIR=$1
+mkdir -p $BDIR
+chgrp postgres $BDIR
+chmod 770 $BDIR
+cd $BDIR
+echo "Creating backup folder $BDIR"
+echo ""
+
+# Conf files
+echo "Backing up config files... "
+tar --selinux -czvf config_files.tar.gz \
+/etc/katello/ \
+/etc/katello-installer/ \
+/etc/candlepin/ \
+/etc/foreman/ \
+/etc/foreman-proxy/ \
+/etc/gutterball/ \
+/etc/hammer/ \
+/etc/pulp/ \
+/etc/puppet/ \
+/etc/qpid/ \
+/etc/qpid-dispatch/ \
+/etc/elasticsearch/ \
+/etc/tomcat6/ \
```

Comment:
On EL7 all things tomcat are `tomcat` and on EL6 they are `tomcat6` so I'd reduce this to `/etc/tomcat*` and update the other tomcat locations in the script.